pkg/plugin/v3/scaffolds/webhook.go

```diff
@@ -75,12 +75,14 @@ func (s *webhookScaffolder) newUniverse() *model.Universe {
 }
 
 func (s *webhookScaffolder) scaffold() error {
+
 	if s.conversion {
 		fmt.Println(`Webhook server has been set up for you.
 You need to implement the conversion.Hub and conversion.Convertible interfaces for your CRD types.`)
 	}
 
-	s.config.UpdateResources(s.resource.GVK())
+	s.config.AddWebhook(s.resource.Resource(),
```

Comment:
So you don't need a new config method:

You can set webhook booleans on `webhookScaffolder` like:
```go
type webhookScaffolder struct {
	config      *config.Config
	boilerplate string
	resource    *resource.Resource

	// Webhook type options.
	webhooks config.Webhooks
}

func NewWebhookScaffolder(
	config *config.Config,
	boilerplate string,
	resource *resource.Resource,
	webhooks config.Webhooks,
) cmdutil.Scaffolder {
	return &webhookScaffolder{
		config:      config,
		boilerplate: boilerplate,
		resource:    resource,
		webhooks:    webhooks,
	}
}
```

Then do:
```suggestion
	res := s.resource.Resource()
	res.Webhooks = s.webhooks
	s.config.UpdateResource(res)
```